reya_trading/auth/signatures.py

```diff
@@ -0,0 +1,332 @@
+"""
```

Comment:
would be nice to have some tests that involve signatures, if smth small is off with signatures then on-chain the recovered address is wrong and it's quite difficult to understand why

Reply:
can be included for cicd tests